### PR TITLE
simplify displaying the executors

### PR DESCRIPTION
**Pull request by stephenc**
follow-up to comment by @jglick to #1596

@reviewbybees

### CODE REVIEW

File: core/src/main/java/hudson/model/Executor.java [hunk 1]
```diff
@@ -587,6 +587,17 @@ public boolean isActive() {
     }
 
     /**
+     * Returns {@code true} if the {@code executorCell.jelly} for this {@link Executor} should be visible.
+     *
+     * @return {@code true} if the {@code executorCell.jelly} for this {@link Executor} should be visible.
```
Comment (olivergondza) [inline core/src/main/java/hudson/model/Executor.java]: This seems redundant.

Comment (stephenc) [inline core/src/main/java/hudson/model/Executor.java]: I don't like leaving the first sentence blank and I don't like a missing `@return`

**Comment by tfennelly** [comment]
:+1:

**Comment by stephenc** [comment]
@jglick if you are :+1: on this please merge it for me ;-)

**Comment by jglick** [comment]
:+1:

File: core/src/main/java/hudson/model/Executor.java (hunk 1 above)
Comment (KostyaSha) [inline core/src/main/java/hudson/model/Executor.java]: But you can wrote description in description and not what is returned

Comment (jglick) [inline core/src/main/java/hudson/model/Executor.java]: Yeah that would be my advice too!

File: core/src/main/java/hudson/model/Executor.java [hunk 2]
```diff
@@ -587,6 +587,17 @@ public boolean isActive() {
     }
 
     /**
+     * Returns {@code true} if the {@code executorCell.jelly} for this {@link Executor} should be visible.
```
Comment (daniel-beck) [inline core/src/main/java/hudson/model/Executor.java]: Should be `iff`.

**Pull request merged**